haystack/components/extractors/named_entity_extractor.py

```diff
@@ -0,0 +1,429 @@
+from abc import ABC, abstractmethod
+from contextlib import contextmanager
+from dataclasses import dataclass
+from enum import Enum, EnumMeta
+from typing import Any, Dict, List, Optional, Union
+
+from ... import ComponentError, DeserializationError, Document, component, default_from_dict, default_to_dict
+from ...lazy_imports import LazyImport
+
+with LazyImport(message="Run 'pip install transformers'") as transformers_import:
```

Comment:
I think to be consistent with the rest of Haystack v2 we should update this to be 
```python
with LazyImport(message="Run 'pip install transformers[torch]'")
```
since the HF pipeline also requires `torch` to run and this is how we have been including the torch dependency for other HF transformer pipelines and models.

Reply:
Good point, fixed!